ShipManagementService.Infrastructure/Database/ShipManagementDbContextFactory.cs

```diff
@@ -0,0 +1,22 @@
+﻿using Microsoft.EntityFrameworkCore;
```

Comment:
You have an DbContextFactory, but you don't use this! Please remove this class or make use of it. (Again stop copy paste code !!! )

Reply:
it is used in the serviceRepository